pi-hole.json

```diff
@@ -1,72 +1,75 @@
                         "ui": true
                     }
                 },
-                "volumes": {
-                    "/etc/pihole": {
-                        "description": "Choose a share for Pi-Hole configuration. Eg: create a Share called pihole-config for this purpose alone.",
-                        "label": "Pi-Hole config"
-                    },
-		    "/etc/dnsmasq.d": {
-			"description": "Choose a share for dnsmasq configuration. Eg: create a Share called dnsmasq-config for this purpose alone.",
-			"label": "dnsmasq config"
-		    }
-                },
-                "environment": {
-                    "ServerIP": {
-                        "description": "Enter IP-adress of rockstor server. If not specified it will default to internal docker IP and it will not work.",
-                        "label": "Rockstor IP",
-                        "index": 1
-                    },
-                    "WEBPASSWORD": {
-                        "description": "Enter desired webpassword for pi-hole.",
-                        "label": "Web-Password",
-                        "index": 2
-                    }
-                }
-            }
-        },
-        "description": "DNS Server that acts as a black hole for Internet advertisements.<p>Based on the official docker image: <a href='https://hub.docker.com/r/pihole/pihole' target='_blank'>https://hub.docker.com/r/pihole/pihole</a>, available for amd64 and arm64 architecture.</p>",
-        "more_info": "<h4>PI-HOLE™: A BLACK HOLE FOR INTERNET ADVERTISEMENTS</h4><p><b>Admin page</b></p><p>To access admin interface go to URL: http://[SERVERIP]/Admin</p><p>If you have different port than 80 you need to specify that in the URL.</p><p><b>Block Over 100,000 Ad-serving Domains</b></p><p>Known ad-serving domains are pulled from third party sources and compiled into one list.</p><p><b>Block Advertisements On Any Device</b></p><p>Network-level blocking allows any device to block ads, regardless of hardware or OS.</p><p><b>Improve Overall Network Performance</b></p><p>Since ads are blocked before they are downloaded, your network will perform better.</p>",
-        "ui": {
-            "slug": "admin"
-        },
-        "volume_add_support": false,
-        "website": "https://pi-hole.net/",
-        "version": "2.0"
-    }
+				"volumes": {
+					"/etc/pihole": {
+						"description": "Choose a share for Pi-Hole configuration. E.g.: create a Share called pihole-config for this purpose alone.",
+						"label": "Pi-Hole config"
+					},
+					"/etc/dnsmasq.d": {
+						"description": "Choose a share for dnsmasq configuration. E.g.: create a Share called dnsmasq-config for this purpose alone.",
+						"label": "dnsmasq config"
+					}
+				},
+				"environment": {
+					"FTLCONF_LOCAL_IPV4": {
+						"description": "Enter IPv4 adress of rockstor server. If not specified, it will default to internal docker IP and it will not work.",
```

Comment:
```suggestion
						"description": "Enter the IPv4 address of the Rockstor host. If not specified, it will default to the internal docker IP and will not work.",
```

Reply:
A minor niggle typo/text change. Not important but if we get to do this in this pr then fine. Otherwise we can follow up after some field testing as it would be great to publish as-is anyway. Given the improvements thus far.